program_info/modrinth-mrpack-mime.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
This file should be called `x-modrinth-modpack.xml`, since that's what it'll be called in `/usr/share/mime/application`